### ISSUE TITLE
Hovering over multi line MD links creates ghosting effect

### ISSUE THREAD

**Issue by sawhney17** [athensresearch/athens#1131]
**Problem**
For links that are multiple lines but are shortened to only fit on one line like this:

`[https://pubmed.ncbi.nlm.nih.gov/30839...](https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqbTlLRm9meWRnMDdrRTgxMmpZZmhrMjlrTWNtZ3xBQ3Jtc0ttQVBzbUFRY1B3M1dPUVByeHlpLU8zZ0xSbWF2YXVmcEU4em9jMDFQNjVaTjRtWFZHQ292UExKQXBBZnVhU1RmWDJVMG9xcjhhaWpBY3ZtLVdLSHh5VlJOMUUtVmJuZW4yRUpWaFR2UHdNc3Zva1BVVQ&q=https%3A%2F%2Fpubmed.ncbi.nlm.nih.gov%2F30839054%2F)`

For these links, that should fit on one line, a weird ghosting effect occurs with the bullet box expanding upon hover. 

Apologies for the vague description, not sure how to describe it, watch the video
**Screenshots/Demo**
https://www.loom.com/share/398d35a9fb8149238f51c644f9790b7f
**Athens Version**
Athens Beta 78

**Comment by ozimos** [comment]
This appears to be a CSS Styling issue and I think I have narrowed it down to https://github.com/athensresearch/athens/blob/main/src/cljs/athens/views/blocks/content.cljs#L34. The style rule restores the textarea to the cssdom on hover. I need to add some styles to keep the invisible text area to one line.  On my dev environment, when I change the css style code I see changes in the electron app. But when I remove those changes from the code, the new additions remain on the app unless I explicitly add new css rules to counter the first set of changes. It is as if the css reloading process only adds changes to the stylesheet and does not reload the entire document stylesheet for every change. Is my Athens dev setup wrong. If so how do I fix.  I followed the setup instructions at https://athensresearch.gitbook.io/handbook/development/development#calva. I am working on Windows using windows subsytem for linux (WSL) so I also use Xming Server http://www.straightrunning.com/XmingNotes/ to display the Athens electron app
